src/cli/init.ts

```diff
@@ -193,27 +201,53 @@ export async function handler(argv: Arguments) {
   // we should ask just in case.
   if (!ts) {
     ts = await confirm({ message: 'Is this a TypeScript project?', default: true });
+  }
 
-    if (ts) {
-      await createTSFile(path.join(process.cwd(), 'video.d.ts'));
-      changes.push([log.add, `Created video.d.ts.`]);
-    }
+  if (ts) {
```

Comment:
moved this out of the `if (!ts) {}` scope which was a bug if using the flag I think